Dockerfile.armv7

```diff
@@ -0,0 +1,41 @@
+FROM multiarch/debian-debootstrap:armhf-jessie
+
+RUN apt-get update && apt-get install -y\
+ bison \
+ build-essential \
+ clang \
+ curl \
+ gperf \
+ libasound2 \
+ libasound2-dev \
+ libcap-dev \
+ libcups2-dev \
+ libdbus-1-dev \
+ libgconf-2-4 \
+ libgconf2-dev \
+ libgnome-keyring-dev \
+ libgtk-3-0 \
+ libgtk2.0-0 \
+ libgtk2.0-dev \
+ libnotify-dev \
+ libnss3 \
+ libnss3-dev \
+ libx11-xcb-dev \
+ libxss1 \
+ libxtst-dev \
+ libxtst6 \
+ python-dbusmock \
+ git \
+ wget \
+ xvfb
+
+# Install node.js
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get update && apt-get install -y nodejs
+
+ADD tools/xvfb-init.sh /etc/init.d/xvfb
+RUN chmod a+x /etc/init.d/xvfb
+ADD tools/run-electron.sh /run-electron.sh
+RUN chmod a+x /run-electron.sh
+
+CMD sh /run-electron.sh
```

Comment:
Not a blocker, but if it's easy... these two Dockerfiles are very similar to each other. Is there a way to extract out the common parts into a shared Dockerfile that each of these use?

Reply:
I don't know of a way to do that, but I'm not a Docker expert.

Reply:
Besides... disk space is cheap!  Denormalization FTW!